src/Contracts/DataModels.ts

```diff
@@ -469,8 +469,16 @@ export interface IMaxUsersPerDbAccountExceeded extends IValidationError {
 }
 
 export interface IPhoenixConnectionInfoResult {
+  /**
```

Comment:
we can just remove them right? there is no backward compat issue in case its removed.